### PR TITLE
[FIX][10.0] travis: Update travis file to new standard

### PR DESCRIPTION
**Pull request by lasley**
Hopefully this fixed runbot issue from #139

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,28 +17,32 @@ addons:
...
-  - "export DISPLAY=:910.0"
-  - "sh -e /etc/init.d/xvfb start"
-
 install:
-  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
-  - printf '[options]\n\nrunning_env = dev' > ${HOME}/.openerp_serverrc
-  - pip install unidecode pyserial pycountry
```
Comment (lasley) [inline .travis.yml:36]: I removed this line intentionally - All requirements should be in `requirements.txt` & I will move these there if any come up red.

File: .travis.yml [hunk 2]
```diff
@@ -15,28 +17,32 @@ addons:
...
-before_install:
-  - "export DISPLAY=:910.0"
-  - "sh -e /etc/init.d/xvfb start"
-
 install:
-  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
-  - printf '[options]\n\nrunning_env = dev' > ${HOME}/.openerp_serverrc
```
Comment (lasley) [inline .travis.yml:35]: I removed this line intentionally - I don't see how it provides functionality to the build. Correct me if I'm wrong.

**Comment by pedrobaeza** [comment]
OK, waiting for a green result for fast-tracking it :wink:

File: .travis.yml (hunk 2 above)
Comment (lasley) [reply .travis.yml:35]: Oh wait no I figured it out, adding back

**Comment by lasley** [comment]
Alright we're 🍏  - I'll squash and port to the other versions

**Comment by pedrobaeza** [comment]
I can squash on merge here...

**Comment by lasley** [comment]
True - just made it so I could cherry pick immediately and get the other ones online quicker 😉

**Comment by pedrobaeza** [comment]
OK, waiting for runbot to finish for merging it.

**Comment by rafaelbn** [comment]
Thanks @lasley !!!

**Pull request merged**